go.mod

```diff
@@ -162,6 +162,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require gopkg.in/yaml.v2 v2.4.0
```

Comment:
this should be replaced with .v3 (some wrong import)

Reply:
I think it was there before me, it just got moved here somehow :)
But I'll fix that for sure